test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -392,8 +391,7 @@ def setUp(self):
         Job.dflt_attributes[ATTR_k] = 'oe'
         # Increase the log level
         a = {'log_events': '2047'}
```

Comment:
Now that we changed the debug messages to debug4, does this need to change?

Reply:
PBSEVENT_DEBUG4 is defined as 0x0800 (which is 2048), so yes, I'll update that.